cfd_protocol/src/lib.rs

```diff
@@ -78,25 +71,112 @@ pub fn build_cfd_transactions(
         maker.lock_amount + taker.lock_amount,
     );
 
+    build_cfds(
+        lock_tx,
+        (
+            maker.identity_pk,
+            maker.lock_amount,
+            maker.address,
+            maker_punish_params,
+        ),
+        (
+            taker.identity_pk,
+            taker.lock_amount,
+            taker.address,
+            taker_punish_params,
+        ),
+        oracle_params,
+        refund_timelock,
+        payouts,
+        identity_sk,
+    )
+}
+
+pub fn renew_cfd_transactions(
+    lock_tx: PartiallySignedTransaction,
+    (maker_pk, maker_lock_amount, maker_address, maker_punish_params): (
+        PublicKey,
+        Amount,
+        Address,
+        PunishParams,
+    ),
```

Comment:
Why does `create_cfd_transactions` use `PartyParams` but `renew_cfd_transactions` doesn't?

Reply:
It thought about it but it would be a bit redundant to pass in the lock PSBT twice.

Reply:
Didn't see that, fair enough :)